ocaml/ocamldoc/odoc_ast.ml

```diff
@@ -487,6 +491,7 @@ module Analyser =
                      )
                    else
                      tt_analyse_method_expression env current_method_name comment_opt ~first: false body
+               | {c_rhs=Pattern_guarded_rhs _} :: [] -> []
```

Comment:
Nit: it helps to have short-cases earlier &mdash; that way, it's easier to see the scrutinee together with the patterns that match it on the same screenful of code. I'd consider moving this up.

Reply:
Makes sense, moved up to just before the last case.